src/main/java/com/suse/saltstack/netapi/config/ClientConfig.java

```diff
@@ -86,5 +86,4 @@ public Key(T defaultValue) {
        Object value = store.get(key);
        return value != null ? (T)value : key.defaultValue;
```

Comment:
I usually use it like i would other unary operators but like you said its just personal preference. I don't have a problem with putting a whitespace there just need to get used to it. :)